content/trust-management/trust-authority.md

```diff
@@ -174,28 +214,53 @@ Where:
-| `B_MISINFORMATION` | If the blacklisted entity is a source, it has been publishing false or unverified information intentionnally. If it's a trust authority, it has been certifying sources publishing false or unverified information as trustworthy intentionnally.                                                                                |
-| `B_ABANDONED`      | The source or trust authority has ceased its activity and/or to publish articles through this federation, therefore the entity isn't used anymore and very unlikely to ever be used again. Blacklisting it then prevents someone else from getting their hands on the entity's keys and tokens and impersonate its former owner. |
+#### `B_COMPROMISED`
+
+This code can have different meaning depending on the target:
+
+| Target type                       | Meaning                                                                                                                                                                                           |
+|:----------------------------------|:--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **Source** or **trust authority** | One of the public keys or the Matrix user of the source or trust authority has been compromised.                                                                                                  |
+| **Node**                          | The node is managed or has been taken over by people trying to harm the federation, impersonate other news sources or taking over Matrix accounts that were previously considered as trustworthy. |
+
+
+
+#### `B_MISINFORMATION`
+
+This code can have different meaning depending on the target:
+
+| Target type         | Meaning                                                                                                                                             |
+|:--------------------|:----------------------------------------------------------------------------------------------------------------------------------------------------|
+| **Source**          | The source has been publishing false or unverified information intentionnally.                                                                      |
+| **Trust authority** | The trust authority has been certifying sources publishing false or unverified information as trustworthy intentionnally.                           |
+| **Node**            | The node exists in the sole purpose of registering sources or trust authority that either post unreliable information or certify it as trustworthy. |
```

Comment:
```suggestion
| **Node**            | The node exists in the sole purpose of hosting sources or trust authorities that either post spam, phishing or unreliable information or certify deceptive entities as trustworthy. |
```

Reply:
Will reword slightly (since spam and fishing already are forms of unreliable information, so there's no use in explicitely mention these).